Libplanet/Net/Protocols/KademliaProtocol.cs

```diff
@@ -255,25 +255,21 @@ public async Task CheckReplacementCacheAsync(CancellationToken cancellationToken
             _logger.Debug("Replacement cache checked.");
         }
 
-#pragma warning disable CS4014
         public void ReceiveMessage(Message message)
         {
             switch (message)
             {
                 case Ping ping:
-                    ReceivePingAsync(ping);
+                    ReceivePing(ping);
                     break;
 
                 case FindNeighbors findPeer:
-                    ReceiveFindPeerAsync(findPeer);
-                    break;
-
-                default:
-                    UpdateAsync(message.Remote);
+                    ReceiveFindPeer(findPeer);
                     break;
             }
+
+            var t = UpdateAsync(message?.Remote);
```

Comment:
if we don't want to `await` for `UpdateAsync()`, how about using `#pragma warning disable CS4014` and writing comment about it?

Reply:
Ok, I'll revert changes.